src/kp/kp_client.py

```diff
@@ -125,14 +128,21 @@ def request_session_token(self):
                 break
 
         self.session_cookie = session_cookie
+        self.isFetchingToken = False
```

Comment:
Bliver aldrig sat til False hvis der er en exception! Måske pak alt in i en try/except og sæt den til False der også.
Samt i  "except requests.exceptions.JSONDecodeError:" blokken